pandas/tests/io/formats/test_to_latex.py

```diff
@@ -142,46 +169,57 @@ def test_to_latex_with_formatters(self):
         }
         result = df.to_latex(formatters=dict(formatters))
 
-        expected = r"""\begin{tabular}{llrrl}
-\toprule
-{} & datetime64 &  float & int &    object \\
-\midrule
-index: 0 &    2016-01 & [ 1.0] & 0x1 &  -(1, 2)- \\
-index: 1 &    2016-02 & [ 2.0] & 0x2 &    -True- \\
-index: 2 &    2016-03 & [ 3.0] & 0x3 &   -False- \\
-\bottomrule
-\end{tabular}
-"""
+        expected = _dedent(
+            r"""
+            \begin{tabular}{llrrl}
+            \toprule
+            {} & datetime64 &  float & int &    object \\
+            \midrule
+            index: 0 &    2016-01 & [ 1.0] & 0x1 &  -(1, 2)- \\
+            index: 1 &    2016-02 & [ 2.0] & 0x2 &    -True- \\
+            index: 2 &    2016-03 & [ 3.0] & 0x3 &   -False- \\
+            \bottomrule
+            \end{tabular}
+            """
+        )
         assert result == expected
 
-    def test_to_latex_multiindex(self):
+    def test_to_latex_multiindex_column_tabular(self):
         df = DataFrame({("x", "y"): ["a"]})
         result = df.to_latex()
-        expected = r"""\begin{tabular}{ll}
-\toprule
-{} &  x \\
-{} &  y \\
-\midrule
-0 &  a \\
-\bottomrule
-\end{tabular}
-"""
-
+        expected = _dedent(
+            r"""
+            \begin{tabular}{ll}
+            \toprule
+            {} &  x \\
+            {} &  y \\
+            \midrule
+            0 &  a \\
+            \bottomrule
+            \end{tabular}
+            """
+        )
         assert result == expected
 
+    def test_to_latex_multiindex_small_tabular(self):
+        df = DataFrame({("x", "y"): ["a"]})
         result = df.T.to_latex()
-        expected = r"""\begin{tabular}{lll}
-\toprule
-  &   &  0 \\
-\midrule
-x & y &  a \\
-\bottomrule
-\end{tabular}
-"""
-
+        expected = _dedent(
+            r"""
+            \begin{tabular}{lll}
+            \toprule
+              &   &  0 \\
+            \midrule
+            x & y &  a \\
+            \bottomrule
+            \end{tabular}
+            """
+        )
         assert result == expected
 
-        df = DataFrame.from_dict(
+    @pytest.fixture
+    def multiindex_frame(self):
```

Comment:
Just as a matter of convention can you move the fixture to the top of the module rather than in between the tests? (though this was nice to keep here for review)

Reply:
I moved the fixtures to the top of the class. If I take one more step in a separate PR, I would split the large test class into multiple classes and distribute relevant fixtures between them.